hack/make-rules/run-e2e-tests.sh

```diff
@@ -25,20 +25,97 @@ source "${YURT_ROOT}/hack/lib/build.sh"
 readonly LOCAL_ARCH=$(go env GOHOSTARCH)
 readonly LOCAL_OS=$(go env GOHOSTOS)
 readonly YURT_E2E_TARGETS="test/e2e/yurt-e2e-test"
-KUBECONFIG=${KUBECONFIG:-${HOME}/.kube/config}
+edgeNodeContainerName="openyurt-e2e-test-worker"
+edgeNodeContainer2Name="openyurt-e2e-test-worker2"
+export KUBECONFIG=$HOME/.kube/config
+goldflags="${GOLDFLAGS:--s -w $(project_info)}"
+gcflags="${GOGCFLAGS:-}"
+goflags=${GOFLAGS:-}
```

Comment:
we will overwrite `goldflags`, `gcflags`, `goflags` in function set_flags, and these 3 variables seems not be used at any other place. So, please remove these unnecessary definitions.

Reply:
will do;